jtravis/src/main/java/fr/inria/spirals/jtravis/entities/Pagination.java

```diff
@@ -0,0 +1,47 @@
+package fr.inria.spirals.jtravis.entities;
+
+public class Pagination {
+    private int limit;
+    private int offset;
+    private int count;
+    private String href;
+    private boolean is_first;
+    private boolean is_last;
+    private Pagination next;
```

Comment:
That's kind of not true: those do only contains offset and limit. They don't have is_first/last next/last/first and so one. Create another class for them.